apps/expo/src/components/timeAgo/TimeAgo.test.tsx

```diff
@@ -1,18 +1,35 @@
 import { render, screen } from '@testing-library/react-native'
+
 import TimeAgo from './TimeAgo'
 
+/**
+ * The function has been generated since we are using snapshots to do our frontend testing.
+ * This snapshot would be outdated in one week if we were to use a specific date in our testing.
+ * To fix this, we will just get the date from exactly 4 weeks ago to see if the TimeAgo component
+ * will return an item with the appropriate text of "4 weeks ago"
+ * Thus, the snapshot will now never be out of date.
+ */
+function getDateFourWeeksAgo(): Date {
+  const currentDate = new Date()
```

Comment:
I have fixed this one in my pr, and it tested throughout the TimeAlgo and the utility function that goes with it. 

You might expect to see merge conflict on the TimeAlgo wise